examples/mlflow_artifacts/docker-compose.yml

```
@@ -54,6 +54,7 @@ services:
       --port 5500
       --artifacts-destination s3://bucket
       --gunicorn-opts "--log-level debug"
+      --serve-artifacts
```

```suggestion
      --serve-artifacts
      --artifacts-only
```

Can we also add `--artifacts-only`?

---

added :)

---

added